frontend/mulighetsrommet-veileder-flate/src/mock/api/endpoints/tiltakHandlers.ts

```diff
@@ -17,6 +17,21 @@ export const tiltakHandlers = [
     return HttpResponse.json(Object.values(mockTiltakstyper));
   }),
 
+  http.get("*/api/v1/intern/tiltakstyper/stotterPameldingIModia", async () => {
+    const tiltakstyperMedPamelding = [
+      "ARBFORB",
+      "ARBRRHDAG",
+      "AVKLARAG",
+      "DIGIOPPARB",
+      "GRUFAGYRKE",
+      "GRUPPEAMO",
+      "INDOPPFAG",
+      "JOBB",
+      "VASV",
+    ];
```

Comment:
Dette skal vel være `Tiltakskode` og ikke `Arenakode`?

Reply:
I backend mappes det fra `Tiltakskode` til `Arenakode` så for å mocke korrekt må det være arenakodene